code/clue.js

```diff
@@ -1,26 +1,135 @@
 // STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
 
-const mrGreen = {
-  firstName: 'Jacob',
-  lastName: 'Green',
-  color: 'green',
-  description: 'He has a lot of connections',
-  age: 45,
-  image: 'assets/green.png',
-  occupation: 'Entrepreneur'
-}
+let suspects = [{
+    title: 'Mr Green',
+    firstName: 'Jacob',
+    lastName: 'Green',
+    color: '#067352',
+    description: 'He has a lot of connections',
+    age: 45,
+    image: 'assets/green.png',
+    occupation: 'Entrepreneur'
+  },
+  {
+    title: ' Mrs White',
+    firstName: 'Delia',
+    lastName: 'White',
+    color: '#FFFFF4',
+    description: 'Usually a frazzled servant',
+    age: '70',
+    image: 'assets/white.png',
+    occupation: 'Housekeeper'
+  },
+  {
+    title: 'Ms Peacock',
+    firstName: 'Elanor',
+    lastName: 'Peacock',
+    color: '#252949',
+    description: 'An elderly, still-attractive, woman',
+    age: '62',
+    image: 'assets/peacock.png',
+    occupation: 'Socialite'
+  },
+  {
+    title: 'Mr Plum',
+    firstName: 'Victor',
+    lastName: 'Plum',
+    color: '#700360',
+    description: 'A quick-witted, professor with a bow tie and glasses',
+    age: '38',
+    image: 'assets/plum.png',
+    occupation: 'professor'
+  },
+  {
+    title: 'Miss Cassandra',
+    firstName: 'Cassandra',
+    lastName: 'Scarlett',
+    color: '#EC0E47',
+    description: 'A femme fatal',
+    age: '27',
+    image: 'assets/scarlet.png',
+    occupation: 'actress'
+  },
+  {
+    title: 'Cln Mustard',
+    firstName: 'Jack',
+    lastName: 'Mustard',
+    color: '#FBBF54',
+    description: 'A dignified dapper and dangerous military man',
+    age: '55',
+    image: 'assets/mustard.png',
+    occupation: 'officer'
+  }
+]
```

Comment:
I love the fact that you wrote the objects this way.  This is the standard way of doing things and skipping the intermediary variables entirely! 💯 

This way you know that you didn't accidentally forget to add one to the array!